### ISSUE TITLE
Fine tune CHGNet with GGA+U calculations using non-standard U

### ISSUE THREAD

**Issue by enricosireci** [CederGroupHub/chgnet#220]
Hi everyone. I am trying to fine-tune the CHGNet pretrained potential with my own GGA+U calculations where I apply the Hubbard correction on Mn. To do that, I am trying to correct the energies to make them compatible with the pretrained potential through MaterialsProject2020Compatibility. However, this seems to work only for Mn U = 3.9 eV, while my calculations were carried out using U = 2.5 eV. Can anyone clarify how I can solve this issue? Thanks!

**Comment by bowen-bd** [comment]
In such case, the MP2020 correction is not compatible and should not be applied.
In this case your fine-tuning label would be incompatible to pre-training label and you need to have more data to fine-tune.
A dataset size > 1000 structures should be fine
